cmd/replay/main.go

```diff
@@ -27,11 +27,14 @@ import (
 )
 
 func getBlockHeaderByHash(hash common.Hash) *types.Header {
-	enc := wavmio.ResolvePreImage(hash)
+	enc, err := wavmio.ResolvePreImage(hash)
+	if err != nil {
+		panic(fmt.Errorf("Error resolving preimage: %w", err))
+	}
 	header := &types.Header{}
-	err := rlp.DecodeBytes(enc, &header)
+	err = rlp.DecodeBytes(enc, &header)
 	if err != nil {
-		panic(fmt.Sprintf("Error parsing resolved block header: %v", err))
+		panic(fmt.Errorf("Error parsing resolved block header: %w", err))
```

Comment:
Prefer errors.Wrap (it maintains the callstack information)

```suggestion
		panic(errors.Wrap(err, "Error parsing resolved block header"))
```

Reply:
Fixed